app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt

```diff
@@ -109,34 +109,40 @@ class BrowserToolbarView(
             view.apply {
                 elevation = TOOLBAR_ELEVATION.dpToFloat(resources.displayMetrics)
 
-                onUrlClicked = {
+                if (!isCustomTabSession) {
+                    display.setUrlBackground(getDrawable(R.drawable.search_url_background))
+                }
+
+                display.onUrlClicked = {
                     interactor.onBrowserToolbarClicked()
                     false
                 }
 
-                browserActionMargin = browserActionMarginDp.dpToPx(resources.displayMetrics)
-
-                urlBoxView = if (isCustomTabSession) null else urlBackground
-                progressBarGravity = if (isCustomTabSession) PROGRESS_BOTTOM else PROGRESS_TOP
```

Comment:
PROGRESS_BOTTOM and PROGRESS_TOP are still in the companion object but seems to be unused now.

Reply:
Removed. :)